modules/index/src/main/java/org/apache/ignite/internal/index/IndexBuilder.java

```diff
@@ -197,13 +201,15 @@ public void scheduleBuildIndexAfterDisasterRecovery(
     /**
      * Stops index building if it is in progress.
      *
+     * @param zoneId Distribution zone ID.
      * @param tableId Table ID.
      * @param partitionId Partition ID.
      * @param indexId Index ID.
      */
-    public void stopBuildIndex(int tableId, int partitionId, int indexId) {
+    // TODO remove unused method
```

Comment:
why? If it is in this ticket, then it should be indicated.

Reply:
This method is not used anywhere. I removed it.